design-system-docs/src/_component_docs/button.md

```diff
@@ -71,20 +71,4 @@ If you are using the `citizens_advice_components` gem, you can call the componen
 
```

Comment:
Should the button component use the new example source in the 'Using with Rails' section? (L61)